fitlins/base.py

```diff
@@ -98,15 +98,16 @@ def second_level(analysis, block, space, deriv_dir):
             # The underlying contrast name might have been added to by a transform
             for option in [in_name] + in_name.split('.'):
                 files = fl_layout.get(contrast=snake_to_camel(option),
-                                      type='stat', space=space, **sub_ents)
+                                      type='effect', space=space, **sub_ents)
                 if files:
                     data.append(files[0].filename)
                     break
-            else:
-                raise ValueError("Unknown input: {}".format(in_name))
+            # else: ### Comment out to skip errors with periods
+            #     raise ValueError("Unknown input: {}".format(in_name))
```

Comment:
Uhh, can't remember exactly now, but I think if my contrast names had periods in it, it would throw an error. We can comment this back in, the intent was to just skip that one instead of crashing.

Reply:
Sure. Just trying to figure out if this is a case we should work around. I suspect we do generally not want to accept arbitrary inputs.